transaction/core/src/range_proofs/mod.rs

```diff
@@ -9,18 +9,24 @@
 
 extern crate alloc;
 use alloc::vec::Vec;
-use bulletproofs_og::RangeProof;
+use bulletproofs_og::{BulletproofGens, PedersenGens as BPPedersenGens, RangeProof};
 use curve25519_dalek::{ristretto::CompressedRistretto, scalar::Scalar};
+use mc_crypto_ring_signature::PedersenGens;
 use merlin::Transcript;
 use rand_core::{CryptoRng, RngCore};
 
 pub mod error;
-use crate::{
-    domain_separators::BULLETPROOF_DOMAIN_TAG,
-    ring_signature::{PedersenGens, BP_GENERATORS},
-};
+use crate::domain_separators::BULLETPROOF_DOMAIN_TAG;
 use error::Error;
 
+lazy_static! {
+    /// Generators (base points) for Bulletproofs.
+    /// The `party_capacity` is the maximum number of values in one proof. It should
+    /// be at least 2 * MAX_INPUTS + MAX_OUTPUTS, which allows for inputs, pseudo outputs, and outputs.
+    pub static ref BP_GENERATORS: BulletproofGens =
+        BulletproofGens::new(64, 64);
```

Comment:
Is it possible to just use the `MAX_INPUTS`/`MAX_OUTPUTS` consts here?

Reply:
I'd rather not change that in this PR, it seems orthogonal to the purpose. Feel free to make a ticket